apps/prairielearn/src/pages/home/home.ejs

```diff
@@ -204,292 +176,11 @@
-                  <i class="fas fa-laptop-code" aria-hidden="true"></i>
-                  Write elements
-                </a>
-              </div>
-            </div> <!-- card -->
-          </div> <!-- card-columns -->
-        </div> <!-- container -->
-      </div> <!-- jumbotron -->
-
-      <div class="jumbotron bg-light rounded-0 mb-0">
-        <div class="container">
-          <h2>Ready to get started as an instructor?</h2>
-          <p class="lead">
-            Start by requesting a course and creating some content. Join the Slack workspace for ad hoc
-            assistance or some random banter!
-          </p>
-          <a class="btn btn-outline-dark btn-lg mr-2 mb-2" href="<%= plainUrlPrefix %>/request_course">
-            <i class="fab fa-wpforms" aria-hidden="true"></i>
-            Request your course
-          </a>
-          <a class="btn btn-outline-dark btn-lg mr-2 mb-2" href="https://prairielearn.readthedocs.io/en/latest/getStarted">
-            <i class="fas fa-laptop-code" aria-hidden="true"></i>
-            Create your content
-          </a>
-          <a class="btn btn-outline-dark btn-lg mb-2" href="https://prairielearn.slack.com">
-            <i class="fab fa-slack" aria-hidden="true"></i>
-            Join our community
-          </a>
-        </div> <!-- container -->
-      </div> <!-- jumbotron -->
-
-      <div class="jumbotron bg-dark rounded-0 text-light text-right mb-0">
-        <div class="container">
-          <h2>Want to contribute?</h2>
-          <p class="lead">
-            PrairieLearn is open source, so anyone can add features or fix bugs!
-          </p>
-          <a class="btn btn-outline-light btn-lg mb-2" href="https://github.com/PrairieLearn/PrairieLearn">
-            <i class="fab fa-github" aria-hidden="true"></i>
-            GitHub
-          </a>
-          <a class="btn btn-outline-light btn-lg ml-2 mb-2" href="https://prairielearn.slack.com">
-            <i class="fab fa-slack" aria-hidden="true"></i>
-            Slack
-          </a>
-          <a class="btn btn-outline-light btn-lg ml-2 mb-2" href="https://prairielearn.readthedocs.io/en/latest/dev-guide">
-            <i class="fas fa-book" aria-hidden="true"></i>
-            Developer guide
-          </a>
-        </div> <!-- container -->
-      </div> <!-- jumbotron -->
-      <% } %> <!-- isAuthenticated -->
     </main>
 
     <% if (config.homepageFooterText && config.homepageFooterTextHref) { %>
-      <footer class="footer font-weight-light text-light <%= isAuthenticated ? 'tiny text-center' : 'small text-right bg-dark py-3' %>">
-        <div class="<%= isAuthenticated ? 'bg-secondary p-1' : 'container' %>">
+      <footer style="font-size: 70%;" class="footer font-weight-light text-light text-center">
+        <div class="bg-secondary p-1">
           <a class="text-light" href="<%= config.homepageFooterTextHref %>"><%= config.homepageFooterText %></a>
         </div>
       </footer>
```

Comment:
The fact that this is no longer shown in the unauthenticated home page (i.e., login or whatever it redirects to) needs to be accounted for, if this is a requirement for a specific jurisdiction as the comments for this config value indicate.

Reply:
Done in 65e4098da.